vizro-ai/src/vizro_ai/utils/helper.py

```diff
@@ -19,6 +19,14 @@ class PlotOutputs:
     code_explanation: Optional[str] = field(default=None)
 
 
+def _get_df_info(df: pd.DataFrame) -> Tuple[str, str]:
```

Comment:
Ok if really no time, but shall we move to sample as well here? That should be a very quick change across the codebase no?